src/excmds.ts

```diff
@@ -3084,6 +3095,14 @@ export async function bmark(url?: string, ...titlearr: string[]) {
         }
         const validpaths = flatten(treeClimber(tree, "")).filter(x => "path" in x)
         title = title.substring(title.lastIndexOf("/") + 1)
+        //technically an initial title string like `Firefox/` can give us a blank title
+        //once we remove the path, so let's fix that
+        if (title == "") {
+            //retrieve title from current tab
+            let currTitle = (await activeTab()).title
+            title = currTitle
+        }
+
```

Comment:
I'm not sure this is your fault, but 

`bmark google.com` gives me the error `# Error: Type error for parameter query (Value must either: be a string value, or .url must match the format "url") for bookmarks.search. ` which is slightly confusing - you could try using the `forceURI` function.